kernel/init.c

```diff
@@ -45,6 +45,9 @@ BUILD_ASSERT(CONFIG_MP_NUM_CPUS == CONFIG_MP_MAX_NUM_CPUS,
 __pinned_bss
 struct z_kernel _kernel;
 
+__pinned_bss
+atomic_t _cpus_active;
```

Comment:
Probably want this underneath CONFIG_MP_MAX_NUM_CPUS > 1, but one word of storage is minor.  Could also move it to the kernel struct and maybe put it under an API for use in PM, but that's minor and we can clean things up whenever.